### PR TITLE
Gherkin and automation for workload sidebar entry for pipeline triggers

### PR DESCRIPTION
**Pull request by sanketpathak**
Epic: https://issues.redhat.com/browse/ODC-6375
Story: https://issues.redhat.com/browse/ODC-6426

Acceptance criteria:

- A webhook url exists for the generated pipeline when user selects "add pipeline" in the import-from-git flow
- User is provided with guidance on how to add the webhook url to their Git repository as a webhook to trigger the pipeline on Git events

**Checks required for approving Epic gherkin scripts PR:**

- [x] Add @epic-number to the scenarios or feature file [e.g: @odc-xxx]
- [x] Add @to-do for automation possible scenarios
- [x] Add @regression or @smoke based on the importance of the scenario
- [x] Update the test scenarios count in [Automation status confluence Report](https://docs.jboss.org/display/ODC/Automation+Status+Report)
- [x] Check for the linter issues by executing `yarn run gherkin-lint` on frontend folder [Skip epic number tags related linter issues]

**Fix**: https://issues.redhat.com/browse/ODC-6457

**Test Setup**:

Cluster should be running on local

Check the TAGS in frontend/packages/topology/integration-tests/cypress.json file be: ""(@pre-condition or @smoke or @regression) and @[odc-6375](https://issues.redhat.com/browse/odc-6375) and not (@manual or @to-do or @un-verified or @broken-test)"",

Command to execute:

Run a cluster locally and execute the command `npm run test-cypress-topology` simultaneously

Execute the pipelines-on-topology.feature file

**Browser**
Chrome 90

**Test Execution Screenshot:**
Screenshot from 2022-01-13 19-13-25

### CODE REVIEW

**Comment by karthikjeeyar** [comment]
Ran locally, works fine
/lgtm

File: frontend/packages/topology/integration-tests/support/step-definitions/topology/pipelines-on-topology.ts [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+When('user clicks on workload {string} to open sidebar', (workloadName: string) => {
+  topologyPage.componentNode(workloadName).click({ force: true });
+  topologyPage.waitForLoad();
+});
+
+Then('user can see {string} section in Resources tab', (heading: string) => {
+  topologySidePane.selectTab('Resources');
+  topologySidePane.verifyTab('Resources');
+  cy.wait(5000);
```
Comment (jrichter1) [inline frontend/packages/topology/integration-tests/support/step-definitions/topology/pipelines-on-topology.ts:33]: Why is there a static wait though?

**Comment by jrichter1** [comment]
now it /lgtm

**Comment by sanketpathak** [comment]
This PR only affects our e2e tests.
/label qe-approved
/label px-approved
/label docs-approved

**Pull request merged**